app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,45 @@
+class BookingsController < ApplicationController
+  layout "layouts/application_user"
+  def new
+    @football_pitch = FootballPitch.find_by id: params[:football_pitch_id]
+    @booking = Booking.new
+    if logged_in?
```

Comment:
user chưa log in vẫn booking dc á e?

Reply:
không được chị ơi, em xử lý ở file new.html.erb rồi ạ
![image](https://github.com/awesome-academy/ruby_naitei19_soccer_management/assets/108230550/3aa9d7b4-17f8-48e8-92b0-c5cc5792a659)